packages/blade/src/_helpers/storybook/Sandbox/Sandbox.native.tsx

```diff
@@ -8,10 +8,10 @@ function Sandbox({ children }: SandboxProps): JSX.Element {
   return <BaseText fontFamily="code">{dedent(children)}</BaseText>;
 }
 
-export const RecipeSandbox = (props: RecipeSandboxProps): JSX.Element => {
+const RecipeSandbox = (props: RecipeSandboxProps): JSX.Element => {
   const codesandboxURL = `https://codesandbox.io/s/${props.codesandboxId}?file=${props.activeFile}`;
   // @ts-expect-error: Link only support `string` type and breaks when we pass a variable
   return <Link href={codesandboxURL}>Open {props.title} Recipe in CodeSandbox</Link>;
 };
 
-export default Sandbox;
+export { Sandbox, RecipeSandbox };
```

Comment:
can ignore: Sandbox and RecipeSandbox sounds bit weird. seems like `Sandbox` is orphan without a prefix